docs/sources/tempo/configuration/_index.md

```diff
@@ -354,6 +354,9 @@ query_frontend:
     [tolerate_failed_blocks: <int>]
 
     search:
+        # Maximum number of outstanding requests per tenant per frontend; requests beyond this error with HTTP 429.
+        # (default: 2000)
+        [max_outstanding_per_tenant: <int>]
```

Comment:
Although `max_outstanding_per_tenant` is documented here:
 * https://grafana.com/docs/tempo/latest/configuration/manifest/
 * https://grafana.com/docs/tempo/latest/operations/backend_search/

it's not documented here: https://grafana.com/docs/tempo/latest/configuration/

So I believe this change fixes that.

Reply:
nice find!